patches/server/0978-add-PlayerCreateMapEvent.patch

```diff
@@ -0,0 +1,41 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: chmodsayshello <chmodsayshello@hotmail.com>
+Date: Tue, 2 May 2023 18:22:35 +0200
+Subject: [PATCH] add PlayerCreateMapEvent
+
+
+diff --git a/src/main/java/net/minecraft/world/item/EmptyMapItem.java b/src/main/java/net/minecraft/world/item/EmptyMapItem.java
+index e92bb09dd3218d5a13e6251bddd0812acadda2be..8b07c86f22299e9c89e70603abf77ef78209ac31 100644
+--- a/src/main/java/net/minecraft/world/item/EmptyMapItem.java
++++ b/src/main/java/net/minecraft/world/item/EmptyMapItem.java
+@@ -6,6 +6,7 @@ import net.minecraft.world.InteractionHand;
+ import net.minecraft.world.InteractionResultHolder;
+ import net.minecraft.world.entity.player.Player;
+ import net.minecraft.world.level.Level;
++import io.papermc.paper.event.player.PlayerCreateMapEvent; //Paper
```

Comment:
Inline the import

Reply:
Sorry, I might be stupid there, but I don't know whats wrong there

Reply:
Use the fully qualified name in-line instead of importing (e.g. `new io.papermc.paper.event.player.PlayerCreateMapEvent()` instead of `new PlayerCreateMapEvent()`)

Reply:
aaagh, thanks, my bad, let me do that really quick